book/code_documentation.md

```diff
@@ -134,14 +134,14 @@ It is easy to forget which parts of code have been commented out and why they ha
 
 You should instead use appropriate control flow (such as `if/else` statements) to determine when these sections should be run. When changes are required between individual runs of your analysis, you should consider [defining these options in a dedicated configuration file](configuration.md).
 
-
 In summary, you should use comments sparingly but purposefully. Make sure comments:
 
 - explain **why** certain things are done, in order to provide context around the decisions that you have made
 - do not echo what your code is already telling the reader
 - are accurate and still relevant after code changes
 
 (docstrings)=
+
```

Comment:
yes, I think you're right. This was inconsistent and VSCode was giving warnings about wanting white space around headers. I double checked that it didn't affect the links. But I think for readability it would be better to remove the extra line for this.

Reply:
Shall I go back through and alter spacing on a fresh branch to take account of this comment and the one below?

Reply:
I'm happy to keep them in here, and looking at my other PR I've been really inconsistent with this too. We should look into an automated linting approach afterwards, but the changes here are improvements towards the standard 👍🏻